pkg/cmd/roachtest/tests/mixed_version_job_compatibility_in_declarative_schema_changer.go

```diff
@@ -62,7 +62,7 @@ func setShortGCTTLInSystemZoneConfig(
 	return h.Exec(r, "ALTER RANGE default CONFIGURE ZONE USING gc.ttlseconds = 1;")
 }
 
-// executeSupportedDDLs tests all stmts supported in V23_1.
+// executeSupportedDDLs tests all stmts supported in V24_2.
```

Comment:
ah nice catch on the comment. even better so we don't have to remember each time would be:
> executeSupportedDDLs tests all stmts which had DSC support added in the previous major version.

Reply:
true lol let me reword